src/lib/Components/ArtworkFilterOptions/__tests__/SortOptions-tests.tsx

```diff
@@ -0,0 +1,22 @@
+import { Theme } from "@artsy/palette"
+import { FakeNavigator as MockNavigator } from "lib/Components/Bidding/__tests__/Helpers/FakeNavigator"
+import React from "react"
+import * as renderer from "react-test-renderer"
+import { SortOptionsScreen as SortOptions } from "../SortOptions"
+
+describe("Sort Options Screen", () => {
+  let mockNavigator: MockNavigator
+
+  beforeEach(() => {
+    mockNavigator = new MockNavigator()
+  })
+
+  it("renders properly", () => {
+    const SortOptionsScreen = renderer.create(
+      <Theme>
+        <SortOptions navigator={mockNavigator as any} />
+      </Theme>
+    )
+    expect(SortOptionsScreen).toMatchSnapshot()
```

Comment:
I tend to find snapshots not super helpful. Doesn't have to be for this PR, but instead could we test a meaningful aspect of the component? (Such as that the correct number of `OptionListItem`s show up)?